src/DetailsView/components/Issues-details-pane.tsx

```diff
@@ -87,9 +87,22 @@ export class IssuesDetailsPane extends React.Component<IssuesDetailsPaneProps, I
 
     private renderSingleIssue(result: DecoratedAxeNodeResult): JSX.Element {
         const issueData: CreateIssueDetailsTextData = {
-            pageTitle: this.props.pageTitle,
-            pageUrl: this.props.pageUrl,
-            ruleResult: result,
+            rule: {
+                description: result.help,
+                id: result.ruleId,
+                url: result.helpUrl,
+                guidance: result.guidanceLinks,
+            },
+            targetApp: {
+                name: 'name',
```

Comment:
used to be `pageTitle/pageUrl` & passed in from a parent, will have to do something similar since `UnifiedResult` doesn't have it